app/components/language-select.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  config: Ember.inject.service(),
+  i18n: Ember.inject.service(),
+  languageOptions: Ember.computed(function() {
+    return Object.keys(this.get('languageMap'));
+  }),
+  onFinish: null,
+  languageMap: Ember.computed(() => {
```

Comment:
@richiethomas `value` is used to set the currently selected value of the drop down.  It is a way of setting which `option` is `selected`.